src/qml/controls/TextButton.qml

```diff
@@ -13,7 +15,7 @@ Button {
     property bool bold: true
     property bool rightalign: false
     padding: 15
-    hoverEnabled: true
+    hoverEnabled: AppMode.isDesktop
```

Comment:
Still seeing the issue with the TextButton, I believe this is because there is no "DEFAULT" state to return to after the "PRESSED" state on Android; and we don't see this issue on desktop so must be some API difference related to Android causing the UI issue.

![Screenshot_20230531_070221](https://github.com/bitcoin-core/gui-qml/assets/23396902/b1c82132-7f7c-4a32-bfdf-e408ab28b5a8)

something along the lines of this should do:

```diff
diff --git a/src/qml/controls/TextButton.qml b/src/qml/controls/TextButton.qml
index 0bfafecde..2d8bcf779 100644
--- a/src/qml/controls/TextButton.qml
+++ b/src/qml/controls/TextButton.qml
@@ -39,6 +39,14 @@ Button {
         }
     }
     states: [
+        State {
+            name: "DEFAULT"; when: !root.pressed && !root.hovered
+            PropertyChanges {
+                target: root
+                textColor: root.textColor
+                bgColor: root.bgColor
+            }
+        },
         State {
             name: "PRESSED"; when: root.pressed
             PropertyChanges {
```